sdk/src/main/java/co/omise/android/models/NetceteraConfig.kt

```diff
@@ -15,7 +14,7 @@ internal data class NetceteraConfig(
     @field:JsonProperty("device_info_encryption_cert_pem") val deviceInfoEncryptionCertPem: String? = null,
     @field:JsonProperty("directory_server_id") val directoryServerId: String? = null,
     @field:JsonProperty("key") val key: String? = null,
-    @field:JsonProperty("message_version") val messageVersion: String = BuildConfig.MESSAGE_VERSION,
+    @field:JsonProperty("messageVersion") val messageVersion: String? = null,
```

Comment:
Why is it `null` by default?

Reply:
we have to provide default values other wise we will have this error:
```kotlin
.InvalidDefinitionException: Cannot construct instance of `NetceteraConfig` (no Creators, like default construct, exist): cannot deserialize from Object value (no delegate- or property-based Creator)
```
 I think we will have to use [this](https://mvnrepository.com/artifact/com.fasterxml.jackson.module/jackson-module-kotlin) to fix it but for now this is how it is done.